src/Sylius/Behat/Context/Api/Shop/ProductVariantContext.php

```diff
@@ -60,4 +69,40 @@ public function theProductOriginalPriceShouldBe(int $originalPrice): void
 
         Assert::same($response['originalPrice'], $originalPrice);
     }
+
+    /**
+     * @Then /^I should see ("[^"]+" variant) is discounted from ("[^"]+") to ("[^"]+") with "([^"]+)" promotion$/
+     */
+    public function iShouldSeeVariantIsDiscountedFromToWithPromotion(
+        ProductVariantInterface $targetVariant,
+        int $originalPrice,
+        int $price,
+        string $promotionName
+    ): void {
+        $response = $this->responseChecker->getResponseContent($this->client->getLastResponse());
```

Comment:
I might be overthinking but in this case isn't the variable should be called "content"?